libs/server/src/lib/telemetry/telemetry-client.spec.ts

```diff
@@ -0,0 +1,89 @@
+// import { TelemetryClient } from './telemetry-client';
```

Comment:
Oh where oh where did all these tests go...

Reply:
oof, should have deleted that >.> telemetry client was a remnant of the message bus based approach I backed out of